src/splitInteger.test.js

```diff
@@ -8,12 +8,30 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(splitInteger()).toBeInstanceOf(Array);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(5, 1)).toStrictEqual([5]);
+  });
+
+  it(`should return "num" with numbers that are as close as possible`, () => {
+    expect(splitInteger(17, 4)).not.toBe([1, 1, 5, 10]);
+  });
+
+  it(`should return integers in array in ascending order`, () => {
+    expect(splitInteger(14, 3)).toStrictEqual([4, 5, 5]);
+  });
+
+  it(`should return integers in array in ascending order`, () => {
+    expect(splitInteger(17, 4)).not.toBe([4, 4, 5, 4]);
+  });
+
+  it(`should return empty array if "sum" is not a number`, () => {
+    expect(splitInteger('abc', 4)).toStrictEqual([]);
+  });
 
+  it(`should return empty array if "part" is not a number`, () => {
+    expect(splitInteger(7, 'a')).toStrictEqual([]);
   });
```

Comment:
I'm not sure about these ones, do we have any requirements about NaN?

Reply:
There was a requirement in the task 'the input to your function will always be valid'.